example/src/AsyncExample.js

```diff
@@ -175,8 +175,10 @@ const styles = StyleSheet.create({
     textDecorationLine: "underline",
     fontWeight: "500",
     marginBottom: 8,
+    color: "black",
   },
   value: {
     textAlign: "left",
+    color: "black",
```

Comment:
Since above the background is overridden from default to white.  It made the text not appear. 

Windows wants to default to black background and white text for me. I am not sure if that is because my windows is running on a 'dark theme'. I tried addressing it in the windows specific parts but was unsuccessful.

This seems to be work, and stay aligned with original design.